src/test/java/net/masterthought/cucumber/generators/integrations/PageIntegrationTest.java

```diff
@@ -132,10 +132,10 @@ public void generatePage_onJenkinsConfiguration_generatesSummaryTableWithBuildNu
         configuration.setRunWithJenkins(true);
         configuration.setBuildNumber("123");
 
-        page = new StepsOverviewPage(reportResult, configuration);
+        page = new StepsOverviewPage();
 
         // when
-        page.generatePage();
+        initPageGeneartor().generatePage(page);
```

Comment:
I don't like this convention as there is no clear what is tested here exactly: `initPageGenerator` or `generatePage` method. Will be happy to stay with previous clear convention

Reply:
Changed